app/views.py

```diff
@@ -71,13 +71,17 @@ def convert_to_readable_time(time):
 
 def display(forecast, frequency):
     if frequency == 'currently':
-        return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.\
+        return '{}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.\
             format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
 
     info = []
     for count, item in enumerate(forecast.data):
-        info.append('time: {}'.format(convert_to_readable_time(item.time)))
-        info.append('temperature: {}'.format(item.temperature))
+        info.append('{}'.format(convert_to_readable_time(item.time)))
```

Comment:
@admiral96 why the '{}'format thing? it's pointless if there's nothing else in the format string. unless you want to just convert the value to string. then you should just use `str()`